config/crossSections13p6TeV.yaml

```diff
@@ -0,0 +1,718 @@
+  unc: 36.67
+QCD_PT_470to600:
+  crossSec: 626.4
+  reference: XSDB QCD_PT-470to600_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:44
+  unc: 2.863
+QCD_PT_600to800:
+  crossSec: 178.6
+  reference: XSDB QCD_PT-600to800_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:45
+  unc: 0.8268
+QCD_PT_800to1000:
+  crossSec: 30.57
+  reference: XSDB QCD_PT-800to1000_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:47
+  unc: 0.1405
+QCD_PT_1000to1400:
+  crossSec: 8.92
+  reference: XSDB QCD_PT-1000to1400_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:26
+  unc: 0.04204
+QCD_PT_1400to1800:
+  crossSec: 0.8103
+  reference: XSDB QCD_PT-1400to1800_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:30
+  unc: 0.003798
+QCD_PT_1800to2400:
+  crossSec: 0.1148
+  reference: XSDB QCD_PT-1800to2400_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:35
+  unc: 0.0005556
+QCD_PT_2400to3200:
+  crossSec: 0.007542
+  reference: XSDB QCD_PT-2400to3200_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:36
+  unc: 3.779e-05
+QCD_PT_3200:
+  crossSec: 0.0002331
+  reference: XSDB QCD_PT-3200_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:43
+  unc: 1.208e-06
+
+ST_s-channel_antitop:
+  crossSec: '0.0'
+  reference: Take this from XSDB for now
+  unc: ''
+ST_s-channel_top:
+  crossSec: '0.0'
+  reference: Take this from XSDB for now
+  unc: ''
+ST_t-channel_antitop:
+  crossSec: '87.2'
+  reference: https://twiki.cern.ch/twiki/bin/view/LHCPhysics/SingleTopNNLORef
+  unc: +0.9 -0.8 (scale) +1.5 -1.3(PDF+alphaS) +1.8 -1.5(Total) +0.6 -0.7(mass) +0.2 -0.2 (Ebeam) +/-0.1 (Integration)
+ST_t-channel_top:
+  crossSec: '145.0'
+  reference: https://twiki.cern.ch/twiki/bin/view/LHCPhysics/SingleTopNNLORef
+  unc: +1.7 -1.1 (scale) +2.3 -1.5(PDF+alphaS) +2.8 -1.9(Total) +1.3 -0.9(mass) +0.4 -0.3 (Ebeam) +/-0.1 (Integration)
+ST_tW_antitop_2L2Nu:
+  crossSec: 36.05
+  reference: XSDB TbarWplusto2L2Nu_TuneCP5_13p6TeV_powheg-pythia8 modifiedOn 2024-04-03 19:52:23
+  unc: 0.01296
+ST_tW_antitop_4Q:
+  crossSec: 36.05
+  reference: XSDB TbarWplusto4Q_TuneCP5_13p6TeV_powheg-pythia8 modifiedOn 2024-04-03 19:52:51
+  unc: 0.01296
+ST_tW_antitop_LNu2Q:
+  crossSec: 36.05
+  reference: XSDB TbarWplustoLNu2Q_TuneCP5_13p6TeV_powheg-pythia8 modifiedOn 2024-04-03 19:53:18
+  unc: 0.01296
+ST_tW_top_2L2Nu:
+  crossSec: 35.99
+  reference: XSDB TWminusto2L2Nu_TuneCP5_13p6TeV_powheg-pythia8 modifiedOn 2024-04-03 19:50:49
+  unc: 0.01292
+ST_tW_top_4Q:
+  crossSec: 35.99
+  reference: XSDB TWminusto4Q_TuneCP5_13p6TeV_powheg-pythia8 modifiedOn 2024-04-03 19:51:12
+  unc: 0.01292
+ST_tW_top_LNu2Q:
+  crossSec: 35.99
+  reference: XSDB TWminustoLNu2Q_TuneCP5_13p6TeV_powheg-pythia8 modifiedOn 2024-04-03 19:51:41
+  unc: 0.01292
+
+TT:
+  crossSec: '923.6'
+  reference: https://twiki.cern.ch/twiki/bin/view/LHCPhysics/TtbarNNLO#Updated_reference_cross_sections
+  unc: +22.6 -33.4 (scale) +/-  22.8 (PDF alphas) -24.6 +25.4 (mass) (to be multiplied by the BR)
+ttHTobb_M125:
```

Comment:
separate different TT process types into separate sections

Reply:
I added some line breaks and changed the order a little, but I'm not sure if thats what you meant